src/main/webapp/dev/js/common/timezone.js

```diff
@@ -50,11 +50,16 @@ const TimeZone = {
 
     /**
      * Updates the specified <select> field with the chosen time zone.
+     * If the chosen time zone is unrecognized, a new option is added for it.
      */
     updateTimeZone($selectElement, timeZone) {
-        if (moment.tz.names().indexOf(timeZone) !== -1) {
-            $selectElement.val(timeZone);
+        if (moment.tz.names().indexOf(timeZone) === -1) {
+            const o = document.createElement('option');
+            o.text = `${timeZone} (please select one of the above options)`;
```

Comment:
For existing sessions that use fixed offset `ZoneId`s. The fixed offset ID will be displayed here as the final option, and the user will have to select a proper regional time zone to be able to save the feedback session (fixed offset `ZoneId`s are disallowed at the action level by the backend). This will be more obvious once I've added a warning to the user to choose another option.